src/community/json-ld/src/main/java/org/geoserver/jsonld/builders/impl/DynamicValueBuilder.java

```diff
@@ -38,10 +41,23 @@ public DynamicValueBuilder(String key, String expression, NamespaceSupport names
         super(key);
         this.namespaces = namespaces;
         if (expression.startsWith("$${")) {
-            this.cql = ExpressionsUtils.extractCqlExpressions(workXpathFunction(expression));
+            // takes xpath fun from cql
```

Comment:
This should be isolated in a more meaningful private method and some comments explaining waht's going on added.

Reply:
applied too